sources/packages/backend/apps/db-migrations/src/sql/Notifications/Create-notifications.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE IF NOT EXISTS notifications (
+    id SERIAL PRIMARY KEY,
+    -- Reference Columns
+    user_id INT REFERENCES users(id) ON DELETE CASCADE,
+    template_id VARCHAR(1000) NOT NULL,
+    message_id INT REFERENCES messages(id) ON DELETE CASCADE,
+    gc_notify_payload jsonb NOT NULL,
+    date_sent timestamp without time zone,
```

Comment:
We have been using `date_sent TIMESTAMP WITH TIME ZONE` instead of `timestamp without time zone`.